notification/docker/docker-compose.yaml

```diff
@@ -6,6 +6,8 @@ services:
         build:
             context: .
             dockerfile: Dockerfile
+        env_file:
+          - path: ../src/.env
```

Comment:
what does this enable you to do exactly ?

Reply:
In the container, this will automatically load the environment variables found in the file

Reply:
Oh this is the follow up of 1b71e1ea5344945b01a4a5e50f74e4a68b74b9af

Reply:
Maybe in those case you can re-open the previous PR or at least mention the commit that was causing the issue so that we can understand quicker

Reply:
> Oh this is the follow up of [1b71e1e](https://github.com/tdameros/42-transcendence/commit/1b71e1ea5344945b01a4a5e50f74e4a68b74b9af)

Yes